container/templates/build-docker-compose.j2.yml

```diff
@@ -18,5 +18,6 @@ ansible-container:
     - {{ base_path }}:/ansible-container/:Z
     {% if with_volumes %}{% for vol in with_volumes %}
     - {{ vol }}{% endfor %}{% endif %}
+    {% if params.playbook %}- {{ params.playbook }}:/ansible-container{{ params.playbook }}:Z {% endif %}
```

Comment:
The example from #76 is `ansible-container build --playbook="/root/ansible/playbook.yml"`. I'm curious if you have tested it? I know you're still working on this, and I'm jumping ahead.

You might also test with "/etc/ansible/playbook.yml", as "/etc/ansible" is a default path used by Ansible for storing content. The point is to test with something owned by 'root' and not the user,  and observe how permissions behave. 

Also, speaking of permissions, @dustymabe needs to review the use of `:Z` here. It's my understanding we may not want to continue using `:Z`. I'm not an SELinux expert, so I may be wrong. He should at least be aware of it though.